### ISSUE TITLE
Scrollbar is overlapped in component selector

### ISSUE THREAD

**Issue by froschdesign** [zendframework/zf-mkdoc-theme#63]
In Firefox 62:
![dropdown](https://user-images.githubusercontent.com/103362/46556150-095e2c00-c8e6-11e8-8609-841579f95186.png)

@thexpand 
Can you look at this? Thanks!

**Comment by thexpand** [comment]
@froschdesign Yes, of course. I'll reply as soon as I find the reason for this and apply a fix as a PR.
